index.html

```diff
@@ -0,0 +1,141 @@
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="campaign" href="#">CC Campaign</a></li>
+            </ul>
+        </div>
+        <a href="#" class="toggle-btn">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </a>
+        <div id="hamburger-menu">
+            <span class="close-btn">&times;</span>
+            <ul class="hamburger-menu">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+      </div>
+    </header>
+    <section id="home">
+        <h3>"Hello! Sharing world"</h3>
+        <h1>Udemy Online Courses 2023</h1>
+        <p class="course-description">One of the best web development course believing in the value of improvement via learning, creating new opportunities for people from over 180 countries is taking place in February</p>
+        <div class="time">2023.02.6(MON)~7(TUS)</div>
+        <p class="course-location">@San Fransisco, Web development and more</p>
+    </section>
+    <section id="about">
+        <div class="main-program">
+            <h4>Main Program</h4>
+            <div class="underline"></div>
+            <div class="flex-container">
+                <div class="programs">
+                    <img src="images/pricon1.png" alt="lecture logo">
+                    <h5>Lecture</h5>
+                    <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </div>
+                <div class="programs">
+                    <img src="images/pricon2.png" alt="Exhibition logo">
+                    <h5>CC Exhibition</h5>
+                    <p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi.</p>
+                </div>
+                <div class="programs">
+                    <img src="images/pricon3.png" alt="forum logo">
+                    <h5>Forum</h5>
+                    <p>Have the time to share your thoughts and opinions with experts for each topic.</p>   
+                </div>
+                <div class="programs">
+                    <img src="images/pricon4.png" alt="workshop logo">
+                    <h5>Workshop</h5>
+                    <p>Try creating your own work using open source than just watching at it.</p>
+                </div>
+                <div class="programs">
+                    <img src="images/pricon5.png" alt="cc ignite logo">
+                    <h5>CC Ignite</h5>
+                    <p>Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+            <h5 class="join">Join Udemy Course 2023</h5>
+            <h5 class="all-work">SEE THE WHOLE PROGRAM</h5>
+        </div>
```

Comment:
- Please stick to the design guideline and fix the heading alignment, follow the color provided, decrease the size between the articles, and decrease the padding around the text. 👍 

<img width="1440" alt="Screenshot 2023-02-09 at 8 54 13 PM" src="https://user-images.githubusercontent.com/25887253/217898424-bbff0d95-be05-42c6-885b-082ad2e33c85.png">
<img width="1216" alt="Screenshot 2023-02-09 at 8 54 33 PM" src="https://user-images.githubusercontent.com/25887253/217898432-662e5db6-6595-4bcf-8794-c3bf2dd23e02.png">


- Also please fix the overlaying text issue on the mobile view.
<img width="319" alt="Screenshot 2023-02-09 at 9 01 55 PM" src="https://user-images.githubusercontent.com/25887253/217900201-14e5b973-c215-4a24-bed8-0ab41d047e59.png">